mne_bids/mne_bids.py

```diff
@@ -570,6 +570,16 @@ def raw_to_bids(subject_id, task, raw_file, output_path, session_id=None,
         raise ValueError('raw_file must be an instance of str or BaseRaw, '
                          'got %s' % type(raw_file))
 
+    if 'eeg' in raw and 'meg' in raw:
```

Comment:
you could even reduce this logic to `if 'meg' in raw` since any and meg sensor present makes it meg. 

also this will fail because not all `meg` files have `eeg`

Reply:
Was going to say "yep, this will fail for some KIT data", then looked at the tests and saw, yep, it failed for KIT data 😆